### ISSUE TITLE
Rigel in Centaurus  should be Riger Kenttaurus

### ISSUE THREAD

**Issue by GoogleCodeExporter** [sky-map-team/stardroid#14]
```
What steps will reproduce the problem?
1. Menu > Search > Centaurus
2. Go to Centaurus, you will find Rigel in Centaurus
3.

What is the expected output? What do you see instead?
The Rigel should be Riger Kenttaurus or Alpha Centauri

What version of the product are you using? On what operating system?
1.6.4

Please provide any additional information below.
The Chinese name is "南门二"
```

Original issue reported on code.google.com by `john.h...@gmail.com` on 19 Sep 2012 at 9:08

**Comment by jaydeetay** [comment]
Thanks for the bug report.  This is now fixed in head.  I added some extra synonyms too.

**Comment by jaydeetay** [comment]
Fixed in the 1.6.5+ releases.
